lib/node_modules/@stdlib/math/base/special/cidentity/src/main.c

```diff
@@ -17,7 +17,6 @@
 */
 
 #include "stdlib/math/base/special/cidentity.h"
-#include <complex.h>
 
```

Comment:
```suggestion
#include "stdlib/complex/float64.h"

```

Missing the header for `stdlib_complex128_t`.